apps/src/netsim/NetSimVisualization.js

```diff
@@ -263,6 +266,14 @@ NetSimVisualization.prototype.onRemoteChange_ = function() {
   this.distributeForegroundNodes();
 };
 
+/**
+ * Called whenever the simplex wire changes state
+ * @private
+ */
+NetSimVisualization.prototype.onWireStateChange_ = function(newState) {
+  this.changeWireState(newState);
+};
```

Comment:
Is there a reason not to move the body of `changeWireState` into this pass-thru function?